src/localization/authStrings.js

```diff
@@ -13,12 +13,13 @@ export const authStrings = new LocalizedStrings({
     repeat_password: 'Please re-enter password',
     user_name: 'User Name',
     email: 'E-mail Address',
+    invalid_username_or_password: 'Invalid username or password',
   },
   fr: {
     logging_in: 'Connexion en cours',
-    login: 'Se connecter or Ouverture de session',
+    login: 'Se connecter',
     password: 'Mot de passe',
-    user_name: "Identifiiant or Nom d'utilisateur",
+    user_name: 'Identifiiant',
```

Comment:
Is this a typo? `Identifiant` maybe? (google translate seems to agree)

Reply:
@DhanyaHerath Yes it is a typo. Should be as you said ie Identifiant

Reply:
oops. fe50743